rootdir/Android.mk

```diff
@@ -40,11 +40,26 @@ LOCAL_MODULE_PATH := $(TARGET_OUT_VENDOR)/etc/init
 include $(BUILD_PREBUILT)
 endif
 
-ifeq ($(TARGET_NEEDS_SENSORS_PDR), true)
+# Note: Sensors PDR can be on ADSP **or** on SDSP
+ifeq ($(TARGET_NEEDS_ADSP_SENSORS_PDR), true)
+ # Let's error out if the developer makes an impossible choice
+ ifeq($(TARGET_NEEDS_SDSP_SENSORS_PDR), true)
```

Comment:
`ifeq(` needs a space otherwise the parser throws up and (erroneously) fails on the following line:

    device/sony/common/rootdir/Android.mk:58: error: extraneous `else'.